how-to/virtualisation/sev-snp.md

```diff
@@ -0,0 +1,49 @@
+# Confidential Computing with AMD
+
+Secure Encrypted Virtualization - Secure Nested Paging (SEV-SNP) are a set of virtualization features available on the latest AMD Epic CPUs (starting from Roma). These features enable *Confidential Computing* which is a way to completely isolate a workload from the hypervisor and the host Operating System.
+Virtual Machines launched with SEV-SNP features enabled have their CPU registers protected from the host OS and their memory encrypted and integrity protected. The memory protection relies on the CPU's Memory Management Unit (MMU) which verifies the integrity of the secure pages and generates per-VM encryption keys to encrypt them and prevent the host OS from reading their content.
+While using Ubuntu as a guest OS on SEV-SNP VMs has been supported since Ubuntu 24.04 LTS, the host enablement (QEMU and OVMF support) was only added later with Ubuntu 25.04.
+
+## Host configuration
+
+AMD SEV-SNP is fully supported on Ubuntu 25.04 and newer. To launch a VM with these features enabled, you need to make sure that SNP is enabled in the firmware settings and that Address-Space Identifiers (ASIDs) are assigned to it. You also want to make sure memory encryption features are enabled. For more details, refer to [AMD's documentation](https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/tuning-guides/58207-using-sev-with-amd-epyc-processors.pdf) and to specific server configuration manuals.
+
+On the host OS, install `qemu-system-x86_64` and launch QEMU with the following parameters:
+
+```bash
+qemu-system-x86_64 \
+  -enable-kvm \
+  -nographic \
+  -machine q35 -smp 6 -m 6G \
+  -drive "if=virtio,format=qcow2,file=disk.img" \
+  -net nic,model=e1000 -net user,hostfwd=tcp::2222-:22 \
+\
```

Comment:
Stray escaped newline?  Unless it is actually needed, perhaps remove to avoid confusion?

Reply:
It's to distinguish between the non sev-snp flags and the specific sev-snp flags. If you think it's more confusing, I can remove.